### PR TITLE
remove reprepro

### LINKED ISSUE #16050: get rid of reprepro
this is a reminder to myself to make the pr to remove reprepro since really we are now just using it as a glorifed way to get the deb into the pool

### PR DESCRIPTION
**Pull request by jessfraz**
fix #16050

ping @tianon

### CODE REVIEW

**Comment by tiborvass** [comment]
LGTM because I don't want you to open more PRs

File: hack/make/release-deb [hunk 1]
```diff
@@ -105,19 +105,17 @@ for dir in contrib/builder/deb/*/; do
...
 	if [ ! -z "$GPG_PASSPHRASE" ]; then
 		dpkg-sig -g "--passphrase $GPG_PASSPHRASE" \
 			-k releasedocker --sign builder "${DEBFILE[@]}"
 	fi
 
-	reprepro -v $options \
-		-S docker-engine -P "$priority" -C "$component" \
-		-b "$APTDIR" includedeb "$codename" "${DEBFILE[@]}"
+	# add the deb for each component for the distro version into the pool
+	cp "${DEBFILE[@]}" "$APTDIR/pool/$component/d/docker-engine/"
```
Comment (tianon) [inline hack/make/release-deb]: Do we need a `mkdir -p` before this to make sure the path exists?

Comment (jfrazelle) [inline hack/make/release-deb]: updated!

**Comment by tianon** [comment]
:metal: LGTM

You've tested this already and it's known to be working, right? :smile:

**Comment by jfrazelle** [comment]
yes i did nummerous times w experimental

**Comment by jfrazelle** [comment]
:innocent:

**Pull request merged**